README.md

```diff
@@ -435,20 +471,32 @@ Does not compute.... code: 2307,severity: error,content: Cannot find module 'com
 
 And the bit after "Does not compute.... " would be red.
 
-#### compilerOptions _(object) (default={})_
+#### compilerOptions
+| Type | Default Value |
+|------|--------------|
+| `object` | `{}`|
 
 Allows overriding TypeScript options. Should be specified in the same format
 as you would do for the `compilerOptions` property in tsconfig.json.
 
-#### instance _(string)_
+#### instance
+| Type | Default Value |
+|------|--------------|
+| `string` | `TODO`|
```

Comment:
Does this option have a default value? I didn't see one in the heading and wanted to ask. 

If not, I can remove that column

Reply:
tagging @johnnyreilly just in case you didn't see this